tutorandroid/templates/android/build/Dockerfile

```diff
@@ -28,27 +29,26 @@ RUN wget --quiet https://dl.google.com/android/repository/commandlinetools-linux
 # Accept licenses
 # https://developer.android.com/studio/command-line/sdkmanager
 # Check target version: https://github.com/edx/edx-app-android/blob/master/constants.gradle
-ARG ANDROID_API_LEVEL=33
+ARG ANDROID_API_LEVEL=34
 RUN yes | /app/android-sdk/cmdline-tools/bin/sdkmanager --sdk_root=${ANDROID_HOME} --install "platforms;android-$ANDROID_API_LEVEL" 1> /dev/null
 
 ###### Checkout code
 FROM base AS code
 # Install android app repo
-ARG ANDROID_APP_REPOSITORY=https://github.com/openedx/edx-app-android.git
-
-# Please add ANDROID_APP_VERSION value in tutor config from release version https://github.com/openedx/edx-app-android/tags
-ARG ANDROID_APP_VERSION={% if OPENEDX_COMMON_VERSION == "master" %}master{% else %}release/{{ ANDROID_APP_VERSION }}{% endif %}
-ADD --keep-git-dir $ANDROID_APP_REPOSITORY#$ANDROID_APP_VERSION /app/edx-app-android
+ARG ANDROID_APP_REPOSITORY=https://github.com/openedx/openedx-app-android.git
+# Always use the 'main' branch
```

Comment:
it would be good if there is link/reference to mention why it must be main. How will it work in redwood release though?

Reply:
We should use ` redwood.master` branch. The corresponding `open-release/redwood.1` will be created as part of the standard release process.
https://github.com/openedx/openedx-app-android/tree/open-release/redwood.master

Reply:
Later, we will probably propose a semver (tutor way) but for now mobile apps have the usual release process.

Reply:
This would be updated in redwood branch of tutor-android.
@Abdul-Muqadim-Arbisoft Please update the PR base branch to nightly.

Reply:
updated